Widgets/WidgetBase/VarmintWidgetChildren.cs

```diff
@@ -81,6 +81,26 @@ public virtual void AddChild(VarmintWidget widget, bool suppressChildUpdate = fa
 
         //--------------------------------------------------------------------------------------
         /// <summary>
+        /// InsertChild
+        /// </summary>
+        //--------------------------------------------------------------------------------------
+        public virtual void InsertChild(VarmintWidget widget, bool suppressChildUpdate = false)
+        {
+            List<VarmintWidget> updatedChildren = new List<VarmintWidget>();
```

Comment:
No need to create a new list.  Modify children directly like this:

children.Insert(0,widget);